met-api/migrations/versions/46490dc02be5_added_new_columns_to_user_table.py

```diff
@@ -56,11 +54,6 @@ def upgrade():
     op.create_index(op.f('ix_user_username'), 'met_users', ['username'], unique=False)
     op.create_foreign_key('user_status_fk', 'met_users', 'user_status', ['status_id'], ['id'])
 
-    # update IDIR users to type STAFF
```

Comment:
is editing an old migration absolutely necessary?

Reply:
if used constants are removed yes!